Kernel/FileSystem/FATFS/Definitions.h

```diff
@@ -9,7 +9,6 @@
 #include <AK/DOSPackedTime.h>
 #include <AK/EnumBits.h>
 #include <AK/Types.h>
-#include <Kernel/Library/KBuffer.h>
 
 namespace Kernel {
 
```

Comment:
> This makes it possible to include this header from userspace.

Only headers in Kernel/API should be included from userspace

Reply:
I would create a Fat definitions file in Kernel/API with the structs you need then, yeah. Not sure if we have the equivalent of the Linux fs headers yet